### DISCUSSION TITLE
Maintain filename as it is when using wikilinks

### DISCUSSION THREAD

**Discussion by ghost** [artempyanykh/marksman#190] [Q&A] [answered]
When I make wikilinks I have these problems: 

- `.` are not recognized. Example: `animals.cat` is converted to `animalscat`.
-  `spaces` are converted to `-`.
-  `--` are converted to `-`.
- `Uppercase` letters are converted to `lowercase`.
- `@` symbol is omitted.
...

I want to maintain the filename as it is with its `spaces, ., --` ... like in obsidian. Is this possible?

**Comment by artempyanykh** [comment]
@bambirombi yes, please check the "Wiki links" section of the README on how to configure this. Let me know if something's still unclear.

**Reply by ghost** [reply, marked as answer]
Yes, sorry. `wiki.style = "file-stem"` works as expected.
